temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` in production code. If you need debugging, consider using logging or other debugging tools that can be disabled or removed easily.